httplib.h

```diff
@@ -4410,7 +4415,69 @@ inline bool load_system_certs_on_windows(X509_STORE *store) {
   return true;
 }
 #endif
+#ifdef __APPLE__
```

Comment:
What about #elif?

Reply:
fixed
used `#elif` instead of a couple of `#ifdef`-s